src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -71,6 +71,53 @@ const enrollHandsontable = new Handsontable(enrollContainer, {
     ],
 });
 
+/**
+ * Function to update the enrollHandsontable cell settings according to a custom renderer.
+ */
+function updateEnrollHandsontableCellSettings(targetRenderer) {
+    enrollHandsontable.updateSettings({
+        cells: () => {
+            const cellProperties = {};
+            cellProperties.renderer = targetRenderer; // uses function directly
+            return cellProperties;
+        },
+    });
+}
+
+/**
+ * Custom renderer to reset any cell styles and tooltips in the current Handsontable instance.
+ */
+/* eslint-disable prefer-rest-params */
```

Comment:
You seem really liberal in disabling linting rules huh. The rules are there for a reason. Can you at least give some justifications to why you are disabling the rule?

Reply:
Actually the reason I'm doing this is because I'm not sure how to do it in the way ESLint requires, so I disabled it temporarily first. Should make a note next time if I'm doing this.